### ISSUE TITLE
Compatibility with node 0.10.x

### ISSUE THREAD

**Issue by LoicMahieu** [brycebaril/client-request#1]
My CI don't want to go green on 0.10.x due to `client-request`. Problem is due to [`require('util').debuglog`](https://github.com/brycebaril/client-request/blob/master/request.js#L15) which has been introduced by 0.11.x

There is a "polyfill" [`node-debuglog`](https://github.com/sam-github/node-debuglog) but adding a dependency for this thing would be really shame.

I don't known why we even tests our code on 0.10 ^^ Maybe it would be great to write it in doc.
Just for your knowledge... I will sleep on it and probably remove this version.

**Comment by LoicMahieu** [comment]
Quick fix when you don't want to drop 0.10 support:

``` js
debuglog = require('debuglog')
util = require('util')
util.debuglog = debuglog
```

**Comment by brycebaril** [comment]
Just saw this! Fixed by making debuglog a noop on 0.10 vs adding a dependency.

Thanks!
